Wiz/wiz-cloud-configuration-findings/ingest/parser.yml

```diff
@@ -0,0 +1,58 @@
+name: wiz-cloud-configuration-findings
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+
+  - name: parsed_date
+    external:
+      name: date.parse
+      properties:
+        input_field: "{{json_event.message.firstSeenAt}}" # TODO: check maybe we should use `analyzedAt`
+        output_field: datetime
+
+  - name: set_ecs_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          event.kind: "alert"
+          event.category: ["vulnerability"]
+          event.type: ["info"]
+          observer.vendor: "Wiz"
+          event.dataset: "Cloud Configuration Finding"
+
+          "@timestamp": "{{parsed_date.datetime}}"
+
+          log.level: "{{json_event.message.severity}}"
+
+          rule.id: "{{json_event.message.rule.id}}"
+          rule.name: "{{json_event.message.rule.name}}"
+
+          wiz.findings.id: "{{json_event.message.id}}"
+          wiz.findings.status: "{{json_event.message.status}}"
+          wiz.findings.result: "{{json_event.message.result}}"
```

Comment:
I think we can also use the value of `result` to compute `event.outcome`:
- PASS -> success
- FAIL -> failure
- ERROR -> failure
- NOT_ASSESSED discards the field